website/pages/docs/commands/namespace/apply.mdx

```diff
@@ -42,3 +42,21 @@ Create a namespace with a quota
 $ nomad namespace apply -description "Prod API servers" -quota prod api-prod
 Successfully applied namespace "api-prod"!
 ```
+
+Remove a quota from a namespace
+
+Quotas cannot be removed from a namespace once they have been applied, only replaced. To effectively remove a quota, create and apply an unlimited quota.
```

Comment:
We don't have a linter for it for some reason (not sure there is one?), but generally we hard-wrap markdown lines.